src/matrix/room/BaseRoom.js

```diff
@@ -404,11 +404,14 @@ export class BaseRoom extends EventEmitter {
     }
 
     async observePowerLevels() {
+        if (this._powerLevelLoading) { await this._powerLevelLoading; }
```

Comment:
new members should be set to null in the constructor, I think this could cause v8 to de-optimize our code if the hidden class doesn't match any more.